pkg/cmd/admin/top/images.go

```diff
@@ -127,7 +127,13 @@ var _ Info = &imageInfo{}
 func (i imageInfo) PrintLine(out io.Writer) {
 	printValue(out, i.Image)
 	printArray(out, i.ImageStreamTags)
-	printArray(out, i.Parents)
+	shortParents := make([]string, len(i.Parents))
+	for i, p := range i.Parents {
+		if len(p) > 17 {
+			shortParents[i] = p[:17] + "..."
+		}
+	}
+	printArray(out, shortParents)
```

Comment:
The shortParents is just the same list, but with shortened ids. It's done only for readability. So instead of long 72 chars ids, you'll see 20 char length.